Lib/test/test_ssl.py

```diff
@@ -1919,9 +1928,13 @@ def test_connect_capath(self):
         ctx.verify_mode = ssl.CERT_REQUIRED
         ctx.load_verify_locations(capath=BYTES_CAPATH)
         with ctx.wrap_socket(socket.socket(socket.AF_INET)) as s:
-            s.connect(self.server_addr)
-            cert = s.getpeercert()
-            self.assertTrue(cert)
+            try:
+                s.connect(self.server_addr)
+                cert = s.getpeercert()
+                self.assertTrue(cert)
+            except ConnectionResetError as e:
+                # sometimes windows throws ConnectionResetError during the handshake
+                sys.stdout.write(repr(e))
```

Comment:
Again

Reply:
removed the try/except including the sys.stdout.write